src/components/CivitaiLink/CivitaiLinkResourceManager.tsx

```diff
@@ -14,6 +14,7 @@ const supportedModelTypes: ModelType[] = [
   'LoCon',
   'Controlnet',
   'Upscaler',
+  'DoRA',
 ];
```

Comment:
Thinking we could replace this with `const supportedModelTypes = Object.values(ModelType);`